http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -595,13 +596,10 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
 
       // Publish a CreateApp message for Back Leo
       createAppV2Message = CreateAppV2Message(
-        workspaceId,
         app.id,
         app.appName,
-        diskResultOpt.flatMap(d => if (d.creationNeeded) Some(d.disk.id) else None),
```

Comment:
Well, currently we raise an error if a `CreateAppV2Message` is sent with a GCP context. We don't really have WSM support for GKE (and the v2 routes require a WSM workspace).

I decided to leave out disk for now since we don't support it in Azure -- eventually though we should add it when it's supported in WSM.

Reply:
For clarity, this is how I think of the app routes:
* v1 routes -- support GKE as today
* v2 routes -- support GKE and AKS, but require a WSM workspace. So for now, only used with Azure.

Reply:
```
I decided to leave out disk for now since we don't support it in Azure -- eventually though we should add it when it's supported in WSM.
```
so disk will always be managed by LZ?

```
v1 routes -- support GKE as today
v2 routes -- support GKE and AKS, but require a WSM workspace. So for now, only used with Azure.
```
yea, thats how I think of it as well...

Reply:
> so disk will always be managed by LZ?

Yeah, today the LZ has a single node + disk. We're not really tracking the disk explicitly though (and we're not passing it to the helm chart).

<img width="1299" alt="image" src="https://user-images.githubusercontent.com/5368863/199539101-a5a4bf86-6fe6-4455-b394-de13d7f46e2d.png">


In general I think we probably need to design how to do disk management on Azure… e.g. I imagine we'd use WSM to provision the disk.

Reply:
disk will be shared by different apps (potentially by different users)?

Reply:
Yeah -- right now, different apps/users will use the same node+disk. 

They will run in different k8s namespaces, and have different pods, PVs, etc. AppSec seemed OK with kubernetes-level isolation to start with. Definitely leaving the door open for more nodepool/disk management in the future (depending on requirements).